api/app/controllers/users/registrations_controller.rb

```diff
@@ -1,23 +1,82 @@
-# frozen_string_literal: true
-
 module Users
   class RegistrationsController < Devise::RegistrationsController
     respond_to :json
 
+    def update
+      load_resource
+      set_prev_unconfirmed_email
+
+      resource_updated = update_resource(resource, account_update_params)
+      log_update(resource_updated)
+
+      yield resource if block_given?
+      handle_resource_update_response(resource_updated)
+    end
+
+    protected
+
+    def update_resource(resource, params)
+      if params[:password].blank? && params[:password_confirmation].blank?
+        resource.update_without_password(params)
+      else
+        resource.update(params)
```

Comment:
Acá no podemos llamar a `super` nomás?

Reply:
originalmente estaba asi, pero me tira "message": "User couldn't be updated successfully. Current password can't be blank" y no le encontre la vuelta mas que esto